README.md

```diff
@@ -42,12 +42,12 @@ export default {
 
 To customize formatting behavior, see the [Configuration](#configuration) section below.
 
-## Using in VS Code
+## Using in VS Code with the Prettier extension
 
 > **Note**
-> If you're using [Astro's VS Code extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode), Prettier and this plugin are already included. Only follow the guide below to format using Prettier's official extension.
+> This section ONLY applies if you want to use Prettier's VS Code extension to format. [The Astro VS Code extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) uses Prettier and this plugin in the background to format, but does not support all of the features of the Prettier extension, such as the toolbar panel showing Prettier's status.
```

Comment:
```suggestion
[The Astro VS Code extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) uses Prettier and this plugin (`prettier-plugin-astro`) to format your code. You will only need to install the VS Code Prettier extension separately for formatting if:

- You are not using Astro's VS Code extension.
- You want to use features of the Prettier extension that not supported by Astro's own VS Code extension, such as the toolbar panel showing Prettier's status.
```
What about something like this?

Reply:
Works for me! One day Prettier's extension will be less confusing to use and I'll remove this entire section, one day.